index.bs

```diff
@@ -6859,8 +6849,8 @@ network.Cookie = {
     size: js-uint,
     httpOnly: bool,
     secure: bool,
-    sameSite: network.SameSite,
     ? expiry: js-uint,
+    sameSite: network.SameSite,
```

Comment:
@lutien nit, this entry should have been moved back given that it stays as required and should be listed before optional ones. Mind a quick follow-up patch?